src/UI/EditorGroupView.re

```diff
@@ -1,40 +1,19 @@
-/*
- * Editor.re
- *
- * Editor component - an 'Editor' encapsulates the following:
- * - a 'tabbar'
- * - an editor surface - usually a textual buffer view
- */
-
```

Comment:
Decided to clean this up as well, as there was a lot of convoluted, deceptive and unused stuff that made it hard to debug an issue I had with ``position(`Absolute)`` overflowing the actual editor area and overlapping the tabs.

Reply:
Thanks for cleaning this up!